src/site/xdoc/filters/suppressionsinglefilter.xml.template

```diff
@@ -58,7 +58,7 @@
         </p>
         <macro name="example">
           <param name="path"
-                 value="resources/com/puppycrawl/tools/checkstyle/filters/suppressionsinglefilter/Example1.txt"/>
+                 value="resources/com/puppycrawl/tools/checkstyle/filters/suppressionsinglefilter/Example1.java"/>
           <param name="type" value="config"/>
```

Comment:
under each block of config, please add one more macro with `<param name="type" value="code"/>`

all examples in html should be as bundle of config and java code.
See other modules examples on how it should looks like and and how to do this.